<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1110">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Путь к папке с данными</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="name_report_line"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Имя отчета связи</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Имя отчета управления</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="management_report_line"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Имя отчета управления небом</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="management_sky_report_line"/>
         </item>
         <item row="0" column="1">
          <widget class="FilePathEdit" name="fpe">
           <property name="dialogMode" stdset="0">
            <enum>FilePathEdit::DIRPATH</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Требуемые суффиксы </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="sfx_s">
           <property name="text">
            <string>s</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="sfx_ro1">
           <property name="text">
            <string>r01</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="sfx_r1">
           <property name="text">
            <string>r1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="sfx_r2">
           <property name="text">
            <string>r2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="sfx_r3">
           <property name="text">
            <string>r3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="sfx_o">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>o</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="sfx_all">
           <property name="text">
            <string>выбрать все</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Информация для составления отчета</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>МЗД АЗДК-1.5 №</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="number_spinBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>ОДС 1 №</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="number_ods_spinBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Время испытаний</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Выставить дату</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Версия прошивки</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="firmware_line"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Интенсивность </string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="intensity_doubleSpinBox"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Влияние величины</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="influence_doubleSpinBox"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Фокус </string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="focus_doubleSpinBox"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Размер изображения звезды </string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QDoubleSpinBox" name="size_doubleSpinBox"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Интерфейс</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Скорость интерфейса</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QComboBox" name="speed_comboBox">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Режим бинирования</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>Включить</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Сторожевой таймер</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QCheckBox" name="checkBox_timer">
           <property name="text">
            <string>Включить</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Длительность накопления</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QSpinBox" name="duration_spinBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string/>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Путь для формирования отчета</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="open_file">
           <property name="text">
            <string>Открыть файл</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="create_button">
           <property name="text">
            <string>Сделать отчет</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QProgressBar" name="progressBar">
           <property name="styleSheet">
            <string notr="true">QProgressBar {
    text-align: center;
}</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
           <property name="textDirection">
            <enum>QProgressBar::TopToBottom</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="FilePathEdit" name="fpe_2">
           <property name="dialogMode" stdset="0">
            <enum>FilePathEdit::FILESAVE</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="to_path_button">
           <property name="text">
            <string>Перейти в каталог</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="LogWidget" name="loger" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilePathEdit</class>
   <extends>QWidget</extends>
   <header>ngts/filepathedit.h</header>
   <addpagemethod>addPage</addpagemethod>
  </customwidget>
  <customwidget>
   <class>LogWidget</class>
   <extends>QWidget</extends>
   <header>log/logw.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
